### DISCUSSION TITLE
[Informationssystem] Terminkalender Schüler / Lehrer

### DISCUSSION THREAD

**Discussion by Kronprinz03** [SE-TINF22B2/G2-Schoolify#22] [User Stories]
### User Story:
Als Schüler / Lehrer möchte ich eine Möglichkeit haben wichtige Termine, wie Klausuren, direkt zu sehen, um immer den Überblick zu behalten und keine zu vergessen.

### Beschreibung:
Der Schüler / Lehrer soll eine Kalenderansicht haben mit allen wichtigen Terminen.
Zu wichtigen Terminen gehören: 
Klausuren, Ferienbeginn / ende, Schulfeiern (z.B. Abiball), sonstige Termine (z.B Exkursion)
In der Kalenderansicht soll zu sehen sein an welchen Tagen des Monats ein Termin ansteht. 
In einer Listenansicht unter dem Kalender sollen die genauen Termine mit Details und Uhrzeit stehen. 
Der Kalender soll klickbar sein, sodass die Listenansicht auf den richtigen Tag springt.

### Akzeptanz-Kriterien:

- [ ] Kalender zeigt an welchen Tag Termine anstehen
- [ ] Listenansicht gibt Auskunft über Details 
- [ ] Durch Klick in Kalender springt Listenansicht an den richtigen Tag

**Extras**:

- [ ] Farbige Markierung von unterschiedlichen Prioritäten und Events

**Comment by tilwbr** [comment]
- Kalender übersicht, die man filtern kann

**Comment by laralngn** [comment]
Meinst du hier eine eigene Seite oder auf der Homepage?

**Reply by Kronprinz03** [reply]
Würde schon sagen das würde Sinn ergeben

**Reply by laralngn** [reply]
Was davon? 😂

**Comment by two-Gee** [comment]
würde die Notizen und Abgaben erstmal rausmachen und wenn als eigene Userstorie erstellen.
